code/modules/mob/living/living_defense.dm

```diff
@@ -323,13 +323,16 @@
 
 
 /mob/living/ratvar_act()
-	if(stat != DEAD && !is_servant_of_ratvar(src) && !add_servant_of_ratvar(src))
-		src << "<span class='userdanger'>A blinding light boils you alive! <i>Run!</i></span>"
-		adjustFireLoss(35)
-		if(src)
-			adjust_fire_stacks(1)
-			IgniteMob()
-		return FALSE
+	if(stat != DEAD && !is_servant_of_ratvar(src))
+		for(var/obj/item/weapon/implant/mindshield/M in implants)
+			qdel(M)
+		if(!add_servant_of_ratvar(src))

```

Comment:
So, fuck the chaplain (makes sense), and the captain because... why do we set the captain AND ONLY the captain on fire. The Head of Security is fine with his ratvar overlord, the scientists making toxin bombs are fine, but the captain is all like... WAIT I LOVE NANOTRASEN SO MUCH